tests/unit/test_document.py

```diff
@@ -648,6 +656,29 @@ def test_export_images(get_bytes_images_mock):
     shutil.rmtree(output_path)
 
 
+def test_export_images_empty_bounding_box(get_bytes_images_mock):
+    output_path = "resources/output/"
+
+    doc = document.Document.from_gcs(
+        gcs_bucket_name="test-directory", gcs_prefix="documentai/output/123456789/0"
+    )
+
+    del (
```

Comment:
Perhaps put these `doc = ...; del(...` lines of code into a factory function, say, `def make_document_without_bbox`, and each test that needs a document of that shape can call the function to get a copy of it.